Marlin/src/module/temperature.cpp

```diff
@@ -3991,7 +3991,7 @@ void Temperature::isr() {
       #endif
 
       bool wants_to_cool = false;
-      float target_temp = -1, previous_temp = 9999;
+      celsius_t target_temp = -1, previous_temp = 9999;
```

Comment:
Probably This should be float

Reply:
It appears to me that the values used here —coming from the integer deg accessors— will always be whole numbers.

Reply:
Target yes, but previous_temp will receive the float from Celsius field.

Reply:
and there's a calc using it with decimal precision: ` if (previous_temp - current_temp < float(MIN_COOLING_SLOPE_DEG_COOLER)) break;`... So, target_temp can be int, but previous_temp should be float.